LambdaTimeline/View Controllers/RecorderViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  RecorderViewController.swift
+//  LambdaTimeline
+//
+//  Created by Lydia Zhang on 5/5/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+class RecorderViewController: UIViewController, AVAudioPlayerDelegate, AVAudioRecorderDelegate {
+    
+    // Mark: - Delegate
```

Comment:
Great job in adding comments to separate the code into sections based off on what they are. I also want to point out that I really like the fact that the code is very well structured throughout the file it makes it easier to read.